modules/help/help_command.py

```diff
@@ -10,9 +10,7 @@ class NewHelpCommand(commands.MinimalHelpCommand):
 
 	def get_ending_note(self):
```

Comment:
You can just delete this function. It would inherit it from its base and would do exactly what it does now.